docs/resp3_features.rst

```diff
@@ -67,3 +67,35 @@ This means that should you want to perform something, on a given push notificati
     >> p = r.pubsub(push_handler_func=our_func)
 
 In the example above, upon receipt of a push notification, rather than log the message, in the case where specific text occurs, an IOError is raised. This example, highlights how one could start implementing a customized message handler.
+
+Client-side caching
+-------------------
+
+Client-side caching is a technique used to create high performance services.
+It exploits the memory available on application servers, servers that are usually distinct computers compared to the database nodes, to store some subset of the database information directly in the application side.
```

Comment:
"Exploit" usually has a negative connotation. It is better to reword it to something like this:

> It utilizes the memory on application servers, typically separate from the database nodes, to cache a subset of the data directly on the application side.

Reply:
Fixed